### PR TITLE
Use endpoint logging / heartbeats

### PR DESCRIPTION
**Pull request by tejohnso**
## Description
Use endpoint logging / heartbeats.

## Motivation and Context
Endpoint logging implementation.

## How Has This Been Tested?
Tested locally with Charles proxy + preview, remotely on a screenconnect display, and also confirmed in editor.

## Release Plan:
- Change widget-common dependency to master after merging [widget-common PR](https://github.com/Rise-Vision/widget-common/pull/167)
- Rebuild + test + merge

### CODE REVIEW

**Review by stulees — APPROVED** [review]
@tejohnso Looks good to me!

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ jobs:
   preconditions:
     working_directory: ~/Rise-Vision/widget-image
     docker: &BUILDIMAGE
-      - image: jenkinsrise/cci-v2-transitional-widgets:0.0.4
+      - image: jenkinsrise/cci-v2-transitional-widgets:0.0.5
```
Comment (stulees) [inline .circleci/config.yml:7]: @tejohnso Do you recommend all widgets get updated to this version now?

Comment (tejohnso) [reply .circleci/config.yml:7]: Yes, if they're going to use latest widget-tester. It uses a later version of node.

Comment (stulees) [reply .circleci/config.yml:7]: @tejohnso Thanks for figuring out that issue!

**Pull request merged**